lib/processors/uglifier.js

```diff
@@ -1,5 +1,6 @@
 const terser = require("terser");
 const copyrightCommentsPattern = /copyright|\(c\)(?:[0-9]+|\s+[0-9A-za-z])|released under|license|\u00a9/i;
+// const EvoResource = require("@ui5/fs").Resource;
```

Comment:
not needed?

Reply:
Well, that's the whole problem with the current flow.
The preload files are created before the debug files are created and the source uglified, so in effect some of the job is done twice (uglify) and this also doesn't work well with external transpilation.

Ideally the flow should allow for

- src (es6 / yolo)
- transpilation (with source map to the original file)
- uglify (with source map resolved to the original file)
- bundling (with source map resolved to the original files)

The easiest way would be to consider
- source
- copy to dist as debug files (saved as is)
- perform transformation(s) and bundling on the -dbg files